extensions/big-data-cluster/src/installer/download.ts

```diff
@@ -0,0 +1,45 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import * as path from 'path';
+import * as stream from 'stream';
+import * as tmp from 'tmp';
+
+import { succeeded, Errorable } from '../errorable';
+
+type DownloadFunc =
+    (url: string, destination?: string, options?: any)
+         => Promise<Buffer> & stream.Duplex; // Stream has additional events - see https://www.npmjs.com/package/download
+
+let download: DownloadFunc;
+
+function ensureDownloadFunc() {
+    if (!download) {
+        const home = process.env['HOME'];
+        download = require('download');
+        if (home) {
+            process.env['HOME'] = home;
```

Comment:
seems the home EnvVariable is getting updated with its original value.

Reply:
Will have to debug to double check the intention behind this function.